resources/views/dashboard/admin/realops/index.blade.php

```diff
@@ -39,7 +39,10 @@
         @foreach($flights as $f)
             <tr>
                 <td>{{ $f->flight_date_formatted }}</td>
-                <td>{{ $f->flight_number }}</td>
+                <td style="width: 125px">
```

Comment:
Inline styles need to be moved to a .scss file. If you don't know how to do this, just ask me and I'll help you.

Reply:
Looks great!